### ISSUE TITLE
Plans for a new release?

### ISSUE THREAD

**Issue by poVoq** [stelic/limitload#6]
Looks like there has been some activity in the repository lately. Any plans for a new release?

Cool would be also to make it compatible to BlenderPanda: https://github.com/Moguri/BlenderPanda
As that would really make art and script contributions much easier. Missions might be even scriptable using the visual node editor from blender.

**Comment by stelic** [comment]
Yes, we are planning a new release for some time now, but it's moving slowly. We want to finish one more notable piece of the game, before this new version.

As for this BlenderPanda, I will check it out. Thanks for the tip.

**Comment by poVoq** [comment]
Great to hear!

Maybe also configure a site on itch.io so that small releases are easier to push to players. They also offer an convenient way to collect donations for an open-source project.

P.S.: The offer to host a player sub-forum on freegamedev.net still stands if there is a need.

**Comment by poVoq** [comment]
Very slow progress it seems :)

Any plans for Render Pipeline use now that it is easy to integrate:
https://www.panda3d.org/blog/january-2018-development-update/

**Comment by stelic** [comment]
Yes, very slow unfortunately :( I am bugging the code guy to speed up this last piece before new version, but he's really short on time.

Will check his opinion about this Render Pipeline.

**Comment by poVoq** [comment]
I see the re-base on Panda 1.10 is ongoing or complete :)

Very nice!

**Comment by stelic** [comment]
Yeah, sort, maybe :) Yes it does work with Panda 1.10 now. Hopefully we soon make AppImage/Flatpak package for Linux too.
